shared/linkify.ts

```diff
@@ -55,6 +61,25 @@ for (const schema of commonSchemes) {
 	linkify.add(schema + ":", "http:");
 }
 
+linkify.add("web+", {
+	validate(text: string, pos: number, self: LinkifyIt.LinkifyIt) {
+		const tail = text.slice(pos);
+
+		if (webplus_scheme_chars.test(tail)) {
+			const offset = tail.match(webplus_scheme_chars)![0].length;
```

Comment:
that assertion isn't valid...

```js
const webplus_scheme_chars = /^[a-z]+:/i;
console.log("web+://invalid.example");
console.log("://invalid.example".match(webplus_scheme_chars));
// web+://invalid.example
// null

```

Reply:
the assertion is valid because it is wrapped in a conditional of `webplus_scheme_chars.test(tail)`

Reply:
Ah, right. Sorry